packages/util-crypto/src/mnemonic/toLegacySeed.ts

```diff
@@ -24,10 +24,17 @@ import { mnemonicValidate } from './validate';
  * }
  * ```
  */
-export function mnemonicToLegacySeed (mnemonic: string, password = '', onlyJs = false): Uint8Array {
-  assert(mnemonicValidate(mnemonic), 'Invalid bip39 mnemonic specified');
-
-  return isReady() && !onlyJs
-    ? bip39ToSeed(mnemonic, password)
-    : mnemonicToSeedSync(mnemonic, password).subarray(0, 32);
+export function mnemonicToLegacySeed(mnemonic: string, password = "", onlyJs = false, isEthereum: boolean): Uint8Array {
+  // IS THIS USED BY ANY OTHER BC THAN ETHEREUM? WE COULD SIMPLIFY THIS FUNCTION IF IT ISNT THE CASE
```

Comment:
Indeed. Actually Ledger as well, but for the time being that does not go via this function (just because is needs 64 bytes, not 32 - so the flag you added is actually what gets used in Ledger as well, but the way to correctly solve it is to have the byteLength passed, not the flag)

Reply:
Ok thanks I will add the byte length.

Have you had the occasion of reviewing my comment above about the sign/verify tests? 

Thanks in advance

Reply:
Bt tricky, since atm if specified it needs to just ignore the wasm path (I actually did add it and then for some weird reason it didn't go into the commit healing with the Ledger HD - my mistake, I was switching between 2 branches there,  guess the final ended up being on the non-changed branch, i.e. it doesn't use this call there)

Reply:
ok